src/Templates/src/templates/maui-blazor/.template.config/localize/templatestrings.fr.json

```
@@ -1,7 +1,7 @@
 {
   "author": "Microsoft",
-  "name": ".NET MAUI Blazor App (Preview)",
-  "description": "A project for creating a .NET MAUI application for iOS, Android, Mac Catalyst, and WinUI using Blazor",
-  "postActions/openInEditor/description": "Opens Pages/Index.razor in the editor.",
-  "symbols/applicationId/description": "Overrides the $(ApplicationId) in the project"
+  "name": "Application .NET MAUI Blazor (préversion)",
+  "description": "Projet de création d’une application .NET MAUI pour iOS, Android, Mac Migrate et WinUI à l’aide de Blazor",
```

looks like they accidentally translated `Catalyst` here
```suggestion
  "description": "Projet de création d’une application .NET MAUI pour iOS, Android, Mac Catalyst et WinUI à l’aide de Blazor",
```

---

@cristianosuzuki77  is possible to add Catalyst as a word to not translate  ?!